src/pages/development/payments-integrations/payment-gateway/request-builder.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Request Builder
+description: Implement complex building strategies using builder composites
```

Comment:
Not critical, but the descriptions in all these topics are not sentences like in other topic files in this repository. I don't know if we are holding dev site metadata to the same standard as the ExL metadata, so just noting here. Not a show-stopper.

Reply:
yes, good question for @jeff-matthews, i wrote some of the descriptions as phrases to keep them concise, since they are meant primarily for SEO etc. i didn't want to bloat every description with "learn how to" or the even longer "in this guide you will learn how to" etc. please let me know if i should go back and extend these to full sentences.

Reply:
We should use the same description metadata standards for all doc repos regardless of publishing destination. Devsite repos don't currently have any validation to enforce it, but that will change soon.